config/deployment/kustomization.yaml

```diff
@@ -2,6 +2,7 @@ resources:
 - ../crd
 - ../rbac
 - template-deployment.yaml
+- https://raw.githubusercontent.com/open-cluster-management-io/api/main/work/v1/0000_00_work.open-cluster-management.io_manifestworks.crd.yaml
```

Comment:
Why do we need this？ I mean the `hypershift-deployment-controller` runs on the ACM hub cluster, and if it is an ACM hub, there must exist the `manifestwork` CRD.

Reply:
I think have this, will make sure ppl who run the operator without using manifest work work, like they want to run the operator on a kind cluster

Reply:
Yeah, that is right, My main concern is that if ACM is already installed, this might overwrite the original manifest CRD, not sure if that's something to consider.